graph/src/buffer_pile.c

```diff
@@ -1,13 +1,11 @@
 #include "buffer_pile.h"
+#include "macros.h"
 #include <stdlib.h>
 
 struct rage_BufferAllocs {
     rage_BufferAllocs * next;
     rage_BufferAllocs * prev;
-    union {
-        rage_BuffersInfo;
-        rage_BuffersInfo info;
-    };
+    RAGE_ANON_UNION(rage_BuffersInfo, info);
```

Comment:
The use of this in an internal c file confused me at first because I'd labelled it as "useful only for the compile flag" and then thought "switching the compile flag will mean you either break compilation or never use the anonymous union anyway.

Talking with @foolswood, the macro has _two_ uses: 1) to make defining this particular kind of anonymous union more compact and 2) to make using it _in headers only_ optional.

Somewhere I'd like a comment to that effect to save going through the slight WTF when I come back to this code ;-)